eza: update to 0.14.0

herbygillot opened the pull request:
###### Verification <!-- (delete not applicable items) -->
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? <!-- Please don't open a new Trac ticket if you are submitting a pull request. -->
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with ~`sudo port -vst install`~ `sudo port -d destroot`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

<!-- Use "skip notification" (surrounded with []) to avoid notifying maintainers -->

herbygillot commented:
@harens @chadcatlett `eza` is a community-maintained fork that continues `exa`. This PR adds a note to `exa` informing users about `eza`.

harens commented on sysutils/eza/Portfile:
```diff
@@ -46,6 +46,9 @@ variant doc description {Build man pages} {
                 -in=${name}.1.md -out=${name}.1"
             system -W ${worksrcpath}/man "${prefix}/bin/go-md2man \
                 -in=${name}_colors.5.md -out=${name}_colors.5"
+            system -W ${worksrcpath}/man "${prefix}/bin/go-md2man \
+                -in=${name}_colors-explanation.5.md \
+                -out=${name}_colors-explanation.5"
```
I don't think `go-md2man` is needed anymore, since the pandoc port now just [installs the binary](https://github.com/macports/macports-ports/blob/master/textproc/pandoc/Portfile) rather than building from source.

harens reviewed (commented):
Thanks @herbygillot for the PR.

> @harens @chadcatlett eza is a community-maintained fork that continues exa. This PR adds a note to exa informing users about eza.

Yes I saw the ticket you quickly resolved :) ([#68134](https://trac.macports.org/ticket/68134)).

Judging by the [exa README](https://github.com/ogham/exa#readme), it seems like eza is the "official" replacement for exa. Rather than updating the notes, it might be worth using `replaced_by` from the obsolete pg.

herbygillot committed:
exa: replaced by eza

herbygillot committed:
eza: update to 0.14.0

- only use pandoc for generating docs
- install eza_colors-explanation man page

herbygillot commented:
> Thanks @herbygillot for the PR.
> 
> > @harens @chadcatlett eza is a community-maintained fork that continues exa. This PR adds a note to exa informing users about eza.
> 
> Yes I saw the ticket you quickly resolved :) ([#68134](https://trac.macports.org/ticket/68134)).
> 
> Judging by the [exa README](https://github.com/ogham/exa#readme), it seems like eza is the "official" replacement for exa. Rather than updating the notes, it might be worth using `replaced_by` from the obsolete pg.

OK, done

harens reviewed (approved):
Thanks @herbygillot! 👍
